ribbon/src/main/java/feign/ribbon/LBClient.java

```diff
@@ -96,7 +96,7 @@ public RequestSpecificRetryHandler getRequestSpecificRetryHandler(
     Request toRequest() {
       return new RequestTemplate()
           .method(request.method())
-          .append(getUri().toASCIIString())
+          .appendDirectly(getUri().toASCIIString())
```

Comment:
Using Request.create() is exactly easier to make an orignal request, thanks for reminding :) . But the old code(use append) has the logic that add the header "Content-Length" to the request. Base on keeping the old logic and code reuse, should we continue to use appendDirectly()?